bundles/asdi/asdi-logo-plugin/instance.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * @class Oskari.asdi.login.BundleInstance
+ */
+Oskari.clazz.define("Oskari.asdi.logo.BundleInstance",
+    function() {
+      this.dialog = null;
+      this.data = null;
+      this._loc;
+    }, {
+        __name : 'asdi-logo-plugin',
+        getName : function () {
+            return this.__name;
+        },
+
+        start: function() {
+          this._loc = Oskari.getLocalization('asdi-logo-plugin', Oskari.getLang());
+          this.getAboutLinkContent();
+          this.createLogoPlugin();
+        },
+
+        getAboutLinkContent: function() {
+          var me = this;
+          jQuery.ajax({
+            url:Oskari.getSandbox().getAjaxUrl('GetArticlesByTag'),
+            data: {
+              tags: 'asdi-about'
+            },
+            success: function (response) {
+              me.data = response;
```

Comment:
Check if there's an array as response.articles, take content.body from the first one in the array if it's available and save it as me.data.